README.md

```diff
@@ -94,6 +94,37 @@ Set `absoluteFilePathsForReporter` to `true` to use absolute file paths in gener
 [getting_started]: http://gruntjs.com/getting-started
 [vnujar]: https://validator.github.io/validator/
 
+### `wrapPath`
+
+Type: `String`
+Default: `null`
+
+Provide a path to wrap html partials for validation. 
+
+```js
+all: {
+  options: {
+    wrapPath: 'config/htmlValidationWrapper.html',
+    ignore: /(attribute|element)\s+“[a-z-]+-[a-z-]+”\s+not\s+allowed/i
```

Comment:
This seems irrelevant to the given example. Am I missing something? If not, let's keep it focused, and remove this line.